### PR TITLE
[KH1] Support two word ListComands

### LINKED ISSUE #62: Support 2 words List Commands

### PR DESCRIPTION
**Pull request by KengHian**
- close #62

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -150,31 +160,36 @@ public void addPerson(Person person) {
...
 
         addressBook.setPerson(target, editedPerson);
+        context = Context.CONTACT;
     }
 
     /**
      * Replaces the target company with an edited version of itself.
      *
-     * @param target Company to be replaced.
+     * @param target        Company to be replaced.
```
Comment (Green-Tea-123) [inline src/main/java/seedu/address/model/ModelManager.java:184]: What's this supposed to be? If it's a tab I think you should remove it.

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 2]
```diff
@@ -143,7 +148,7 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     /**
      * Tests the Logic component's handling of an {@code IOException} thrown by the Storage component.
      *
-     * @param e the exception to be thrown by the Storage component
+     * @param e               the exception to be thrown by the Storage component
```
Comment (Green-Tea-123) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:151]: same weird spacing here that can be removed.

**Review by Green-Tea-123 — APPROVED** [review]
Just some small cosmetic things, otherwise LGTM.

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (KengHian) [reply src/main/java/seedu/address/model/ModelManager.java:184]: image

It is the auto reformat in my IDE to align parameter description. I don't think it violates any coding standards, but I am more than happy disable that setting, just let me know.

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 2 above)
Comment (KengHian) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:151]: image

Same thing here.

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (Green-Tea-123) [reply src/main/java/seedu/address/model/ModelManager.java:184]: I guess it's fine as long as the spacing isn't unintentional, feel free to resolve the two convos

**Review by Quasant — APPROVED** [review]
LGTM, I never scrutinise the tests though.

**Pull request merged**